packages/xsnap/makefiles/lin/xsnap.mk

```diff
@@ -0,0 +1,129 @@
+ : %.c
+%.o : %.c
+
+GOAL ?= debug
+NAME = xsnap
+ifneq ($(VERBOSE),1)
+MAKEFLAGS += --silent
+endif
+
+HERE = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
@dckc For reasons I can’t explain, I had to replace PWD with HERE below. PWD worked fine when running (cd makefiles/mac && make) at the CLI, but not when running under a Node.js build script (to appear in the next PR). There might be a Node.js heisenbug.

Reply:
Boring. oh well.

But... rather than just telling me in an ephemeral github comment, do you suppose we should tell future-us in a code comment? I can imagine forgetting why it's this way, changing it back to `$(PWD)`, and wasting a bunch of time.